.github/workflows/docker-hub.yml

```diff
@@ -0,0 +1,51 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+# GitHub recommends pinning actions to a commit SHA.
+# To get a newer version, you will need to update the SHA.
+# You can also reference a tag or branch, but the action may change without warning.
+
+name: Publish Docker image
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  push_to_registry:
+    name: Push Docker image to Docker Hub
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v3
+
+      -
+        name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+
+      -
+        name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+      
+      - name: Log in to Docker Hub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
```

Comment:
this requires project admin privileges. Is a matter of creating an account in docker hub, free project is sufficient.

We can add collaborators in Docker Hub, so a dedicated account can be created rather than the primary one.

Once the account has been created, is just a matter of copy&paste credentials as part of the secrets of the project

Reply:
@janiversen I don't think we need to document the concrete user you have created, unless you want this to be known by others in the public (I don't recommend that).
Otherwise, I would recommend avoid any specific reference to a given user. Is something the administrators of the package can decide to manage privately

Reply:
replying to this https://github.com/riptideio/pymodbus/pull/1132#issuecomment-1287645811